mods/gen1/README.md

```diff
@@ -48,8 +48,8 @@ All moves but Swift and Bide (while on duration, not first hit) have a 1/256 cha
 
 Partial Trapping Moves
 ----------------------
-Partial trapping moves let eiher Pokémon switch but target will be unable to move for its duration.
+Partial trapping moves let either Pokémon switch but target will be unable to move for its duration.
 
 TODO. Glitches not implemented
 ------------------------------
-There are a couple of divisions by zero in the original game in the cartridge. Those have not been implemented.
\ No newline at end of file
+There are a couple of divisions by zero in the original game in the cartridge. Those have not been implemented.
```

Comment:
That doesn't mean nothing changed, it means something invisible has changed. Possibly a hidden character, possibly a line ending. I'm going to merge on the assumption that it's harmless.